checkov/common/output/ai.py

```diff
@@ -70,24 +93,38 @@ async def _chat_complete(self, record: Record) -> None:
             return
 
         try:
-            completion = await openai.ChatCompletion.acreate(  # type:ignore[no-untyped-call]
-                model=OPENAI_MODEL,
-                messages=[
-                    {"role": "system", "content": "You are a security tool"},
-                    {
-                        "role": "user",
-                        "content": "".join(
-                            [
-                                f"fix following code, which violates checkov policy '{record.check_name}':\n",
-                                *[line for _, line in record.code_block],
-                            ]
-                        ),
-                    },
-                    {"role": "user", "content": "Explain"},
-                ],
-                temperature=0,
-                max_tokens=OPENAI_MAX_TOKENS,
-            )
+            # define common messages array
+            messages = [
+                {"role": "system", "content": "You are a security tool"},
+                {
+                    "role": "user",
+                    "content": "".join(
+                        [
+                            f"fix following code, which violates checkov policy '{record.check_name}':\n",
+                            *[line for _, line in record.code_block],
+                        ]
+                    ),
+                },
+                {"role": "user", "content": "Explain"},
+            ],
+            # depends on api_type, call ChatCompletion differently
+            logging.info(f"[_chat_complete]: self._api_type: {self._api_type}")
+            if (self._api_type == 'azure'):
+                completion = await openai.ChatCompletion.acreate(  # type:ignore[no-untyped-call]
+                    engine=self.AZURE_OPENAI_DEPLOYMENT_NAME,
+                    messages=messages[0],
+                    temperature=0,
+                    max_tokens=OPENAI_MAX_TOKENS,
+                )
+            else:
+                completion = await openai.ChatCompletion.acreate(  # type:ignore[no-untyped-call]
+                    model=OPENAI_MODEL,
+                    messages=messages[0],
+                    temperature=0,
+                    max_tokens=OPENAI_MAX_TOKENS,
+                )
+
+            logging.info(f"[COMPLETION]{completion}")
```

Comment:
```suggestion
            logging.debug(f"OpenAI request returned: {completion}")
```
or something similar to have a direct context. Also debug level is more than enough.